store_data.rb

```diff
@@ -0,0 +1,60 @@
+def store_data(app)
+  store_people(app)
+  store_books(app)
+  store_rentals(app)
+end
+
+def store_people(app)
+  people = []
+  storage = Set.new
```

Comment:
- [ ] I ran your app and created a person, book, and rental. Now when I try to quit the app using option number 7, the app crashes, and the details I added are not stored in the JSON files. Kindly look into that.

  ![image](https://github.com/Rifat8080/school-library/assets/54848363/c2e7fa36-0321-4d26-8ba1-944e08c6fb08)

- [ ] Since I could not successfully save data in the JSON, I could not verify if the JSON files were created when they did not exist. Kindly consider that scenario as well. When JSON files do not exist and the user creates data the JSON files should be generated and the data saved there, without app crashing.